### ISSUE TITLE
kubernetes-e2e-gce-slow node resize failing

### ISSUE THREAD

**Issue by yujuhong** [kubernetes/kubernetes#19965]
From kubernetes-e2e-gce-slow/1534:

The node controller panics every time it tries to process the unready node:

```
kube-controller-manager.log:

I0121 21:08:28.895874       6 nodecontroller.go:745] node e2e-slow-minion-guc6 hasn't been updated for 45.256594055s. Last out of disk condition is: &{Type:OutOfDisk Status:Unknown LastHeartbeatTime:2016-01-21 21:07:42 +0000 UTC LastTransitionTime:2016-01-21 21:08:23 +0000 UTC Reason:NodeStatusUnknown Message:Kubelet stopped posting node status.}
E0121 21:08:28.988419       6 util.go:76] Recovered from panic: "invalid memory address or nil pointer dereference" (runtime error: invalid memory address or nil pointer dereference)
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/util.go:70
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/util.go:58
/usr/src/go/src/runtime/asm_amd64.s:402
/usr/src/go/src/runtime/panic.go:387
/usr/src/go/src/runtime/panic.go:42
/usr/src/go/src/runtime/sigpanic_unix.go:26
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/cloudprovider/providers/gce/gce.go:1886
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/cloudprovider/providers/gce/gce.go:1497
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/controller/node/nodecontroller.go:508
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/controller/node/nodecontroller.go:232
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/util.go:120
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/util.go:121
/usr/src/go/src/runtime/asm_amd64.s:2232
```

This is probably caused by #17919

**Comment by alex-mohr** [comment]
Closed by #19967

**Comment by alex-mohr** [comment]
... thanks Yuju for discovering the issue and its cause -- and thanks Justin for getting it turned around so fast!
